### PR TITLE
feat(projects): adding skaffold.dev to pantry

### PR DESCRIPTION
**Pull request by vkumbhar94**
[skaffold.dev](https://skaffold.dev/) facilitates continuous development for container based & Kubernetes applications

### CODE REVIEW

**Comment by vkumbhar94** [comment]
@jhheider @mxcl, this is my first contribution to pkgxdev pantry :-). Please have a look and let me know if I need to do something more.

File: projects/skaffold.dev/package.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+distributable:
+  url: https://github.com/GoogleContainerTools/skaffold/tree/v{{version}}
+
+versions:
+  github: GoogleContainerTools/skaffold/tags
+
+platforms:
+  - darwin
+  - linux
```
Comment (jhheider) [inline projects/skaffold.dev/package.yml]: Platforms key can be omitted if all platforms build/work.

File: projects/skaffold.dev/package.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+dependencies:
```
Comment (jhheider) [inline projects/skaffold.dev/package.yml]: Empty keys should be omitted.

File: projects/skaffold.dev/package.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+build:
+  dependencies:
+    curl.se: '*'
+  working-directory: ${{prefix}}
+  script: |
+    curl -Lfo skaffold curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v{{version}}/skaffold-$PLATFORM
```
Comment (jhheider) [inline projects/skaffold.dev/package.yml]: Does this work correctly? It looks malformed.

File: projects/skaffold.dev/package.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    # old releases are not having github release assets so using storage.googleapis.com for more reliability
+    #curl -Lfo skaffold https://github.com/GoogleContainerTools/skaffold/releases/download/v{{version}}/skaffold-$PLATFORM
+    chmod +x skaffold
+    mkdir -p bin
+    mv skaffold bin
```
Comment (jhheider) [inline projects/skaffold.dev/package.yml]: If this just downloading a blob into place then:

- distributable should just be ~ at the top level.
- it should have a vendored warning key.
```yaml
distributable: ~

warnings:
  - vendored
```

**Review by jhheider — COMMENTED** [review]
Looks good. A few style issues, but green CI tests is very positive.

File: projects/skaffold.dev/package.yml (hunk 3 above)
Comment (vkumbhar94) [reply projects/skaffold.dev/package.yml]: Good catch! How does the gh ci pass - strange!

**Comment by vkumbhar94** [comment]
Incorporated all review comments 😊

**Comment by jhheider** [comment]
LGTM. thanks for the contribution!

**Comment by mxcl** [comment]
Thanks for the contribution! looks like a neat project.

**Pull request merged**